conf/messages

```diff
@@ -46,6 +47,7 @@ common.update = Update
 common.use = use
 common.yes = Yes
 employment.caption = Employment for 6 April {0} to 5 April {1}
+payeEmployment.caption = PAYE employment for 6 April {0} to 5 April {1}
```

Comment:
I would move payeEmployment.caption to go inside the employment information section and rename the key to something like: employment.employerInformation.caption
At present this text is only for that page so it should be page related I think.

Reply:
sure

Reply:
there are other pages that will use this caption so I'll leave it where it is